features/comments_and_kudos/inbox.feature

```diff
@@ -49,6 +49,25 @@ Feature: Get messages in the inbox
       And I press "Filter"
     Then I should not see "guest on Down for the Count"
 
+  Scenario: I can bulk edit comments in my inbox by clicking 'Select'
+    Given I am logged in as "boxer" with password "10987tko"
+      And I post the work "The Fight"
+      And I set my preferences to turn on messages to my inbox about comments
```

Comment:
You can skip this step, as the preference is on my default.

Reply:
Resolved in 168a07ecc